onelogin_aws_cli/__init__.py

```diff
@@ -75,6 +79,26 @@ def get_saml_assertion(self):
 
         self.saml = saml_resp
 
+    def get_ip_address(self):
+        # if ip address has been hard coded in config file, use that
+        ip_address = self.config.get('ip_address')
```

Comment:
Have you thought about adding it into the cli args too, so that we can pull these values in through environment variables? I can think of a couple of cases where it might be more accurate to get the IP aside from making HTTP calls (eg, using VPNs or proxies for traffic).